pages/index.vue

```diff
@@ -254,6 +250,46 @@ export default class extends QiskitPage {
 @import '~carbon-components/scss/globals/scss/typography';
 
 .landing-page {
+  background-color: $white-background-ui;
+
+  &__presentation-container {
+    position: relative;
+    padding-top: $layout-05;
+    padding-bottom: $layout-05;
+    background-attachment: fixed;
+    background-size:
+      2rem 2rem,
+      2rem 2rem;
+    background-image:
+      linear-gradient(to right, $cool-gray-10 1px, transparent 1px),
+      linear-gradient(to bottom, $cool-gray-10 1px, transparent 1px);
+    background-position:
+      -1rem center,
+      -1rem center;
+    background-repeat:
+      repeat,
+      repeat;
+
+    overflow: hidden;
+  }
+
+  &__presentation-picture {
```

Comment:
Could you take a look to picture styles? When I scroll to the right, I get this:
<img width="1679" alt="Screen Shot 2020-05-12 at 12 48 36" src="https://user-images.githubusercontent.com/17231966/81675485-31612a00-944f-11ea-8896-cf2caf7ca07c.png">